lib/utils/helpers.js

```diff
@@ -0,0 +1,16 @@
+const c = require('../constants');
```

Comment:
I'm not a fan of this sort of generic "helper" modules (specially when this has a single function). This function seems to belong together with the objects in the constants module (which could be renamed to reflect that common responsibility.